openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go

```diff
@@ -22,7 +22,7 @@ func newOpenshiftAPIServiceReachabilityCheck(ipForKubernetesDefaultService net.I
 	return newAggregatedAPIServiceReachabilityCheck(ipForKubernetesDefaultService, "openshift-apiserver", "api")
 }
 
-func newOAuthPIServiceReachabilityCheck(ipForKubernetesDefaultService net.IP) *aggregatedAPIServiceAvailabilityCheck {
+func newOAuthAPIServiceReachabilityCheck(ipForKubernetesDefaultService net.IP) *aggregatedAPIServiceAvailabilityCheck {
```

Comment:
I'd like to see the cosmetic changes excluded to reduce the potential for conflicts as we maintain this patch over the years.

Reply:
This one was my doing, the rest were `go fmt` related. I'm happy to revert cosmetic changes, would you like both this and the `go fmt` changes reverted?

Reply:
I don't see any output if I run gofmt -d on either of the other files. Is it me? Maybe your editor is set up to run a gofmt-alike? I'd remove all of the cosmetic changes.

Reply:
What version of Go are you running?

Reply:
I ran `GOTOOLCHAIN=go1.23.4 go fmt openshift-kube-apiserver/openshiftkubeapiserver/patch.go` on master and it didn't make any changes.

Reply:
Cosmetic changes reverted

Reply:
Will rebase to roll up the commit that reverts it in a bit.